### PR TITLE
Refactor existing data type converters to reduce duplication

### PR DESCRIPTION
**Pull request by chr1sjf0x**
@stephenh - I tried refactoring the helpers for converting between db types => ts types => graphql types. I generally like this approach, but I'm getting a strange error. Pushing it up here to see if the same problem happens on CircleCI.

Just now realizing it might be `joist-codgen` => `joist-codegen`?
```
Error: Cannot find module 'joist-codgen'
Require stack:
- /Users/homebound/code/joist-ts/packages/graphql-codegen/build/graphqlUtils.js
- /Users/homebound/code/joist-ts/packages/graphql-codegen/build/generateEnumsGraphql.js
- /Users/homebound/code/joist-ts/packages/graphql-codegen/build/index.js
- /Users/homebound/code/joist-ts/packages/codegen/build/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1080:15)
    at Function.Module._load (internal/modules/cjs/loader.js:923:27)

```

### CODE REVIEW

File: packages/graphql-codegen/tsconfig.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "baseUrl": "./src",
     "paths": {
       "joist-utils": ["../../utils/src"],
-      "joist-codgen": ["../../codegen/src"]
+      "joist-codegen": ["../../codegen/src"]
```
Comment (chr1sjf0x) [inline packages/graphql-codegen/tsconfig.json:18]: @stephenh - I presume this was a typo? Let me know if there is some reasoning for this. It resulted in more diffs than I was expecting in the Codegen files.

File: packages/graphql-codegen/src/graphqlUtils.ts [hunk 2]
```diff
@@ -137,3 +139,20 @@ function mergeDocs(existingDoc: DocumentNode, newDocs: [string, DocumentNode][])
...
+      return "Boolean";
+    case "number":
+      return "Int";
+    default:
+      return type;
+  }
+}
+
+export function mapSimpleDbTypeToGraphQLType(type: string): string | SymbolSpec {
+  return mapTypescriptTypeToGraphQLType(mapSimpleDbTypeToTypescriptType(type));
```
Comment (stephenh) [inline packages/graphql-codegen/src/graphqlUtils.ts:157]: Thanks!

File: packages/graphql-codegen/tsconfig.json (hunk 1 above)
Comment (stephenh) [reply packages/graphql-codegen/tsconfig.json:18]: Oh wow, yep, good fix.

File: packages/integration-tests/src/entities/AuthorCodegen.ts [hunk 3]
```diff
@@ -1,51 +1,51 @@
 import {
+  Flavor,
```
Comment (stephenh) [inline packages/integration-tests/src/entities/AuthorCodegen.ts:2]: Fwiw I think this part of the diff is that I snuck in `prettier-organize-imports-plugin` in the last commit, so if you rebase / `yarn install` / run prettier, these `import` diffs should go away...

**Review by stephenh — APPROVED** [review]
Thanks for the follow up!

**Pull request merged**